docs/references/backend/verify-token.mdx

```diff
@@ -25,74 +25,72 @@ function verifyToken: (token: string, options: VerifyTokenOptions) => Promise<Jw
 1. If the token is valid, the response contains the verified token.
 
-<Tabs items={["Next.js"]}>
-  <Tab>
-  ```ts title="app/api/verify-token/route.ts"
-  import { verifyToken } from '@clerk/backend';
-  import { cookies } from 'next/headers';
+```ts
+import { verifyToken } from '@clerk/backend';
+import { cookies } from 'next/headers';
+
+export async function GET(request: Request) {
+  const cookieStore = cookies()
+  const sessToken = cookieStore.get('__session')?.value
+  const bearerToken = request.headers.get('Authorization')?.replace('Bearer ', '');
+  const token = sessToken || bearerToken;
+
+  if (!token) {
+    return Response.json({ error: 'Token not found. User must sign in.' }, { status: 401 });
+  }
 
-  export async function GET(request: Request) {
-    const cookieStore = cookies()
-    const sessToken = cookieStore.get('__session')?.value
-    const bearerToken = request.headers.get('Authorization')?.replace('Bearer ', '');
-    const token = sessToken || bearerToken;
+  try {
+    const verifiedToken = await verifyToken(token, {
+      jwtKey: process.env.CLERK_JWT_KEY,
+    });
 
-    if (!token) {
-      return Response.json({ error: 'Token not found. User must sign in.' }, { status: 401 });
+    // Check if the token is expired or not yet valid
+    const currentTime = Math.floor(Date.now() / 1000);
+    if (verifiedToken.exp < currentTime || verifiedToken.nbf > currentTime) {
+      throw new Error("Token is expired or not yet valid");
     }
 
-    try {
-      const verifiedToken = await verifyToken(token, {
-        jwtKey: process.env.CLERK_JWT_KEY,
-      });
-
-      // Check if the token is expired or not yet valid
-      const currentTime = Math.floor(Date.now() / 1000);
-      if (verifiedToken.exp < currentTime || verifiedToken.nbf > currentTime) {
-        throw new Error("Token is expired or not yet valid");
-      }
-
-      // Check if the token is issued by a permitted origin
-      const permittedOrigins = ["http://localhost:3000", "https://example.com"]; // Replace with your permitted origins
-      if (verifiedToken.azp && !permittedOrigins.includes(verifiedToken.azp)) {
-        throw new Error("Invalid 'azp' claim");
-      }
-
-      return Response.json({ verifiedToken });
-    } catch (error) {
-      return Response.json({ error: 'Token not verified.' }, { status: 401 });
+    // Check if the token is issued by a permitted origin
+    const permittedOrigins = ["http://localhost:3000", "https://example.com"]; // Replace with your permitted origins
+    if (verifiedToken.azp && !permittedOrigins.includes(verifiedToken.azp)) {
+      throw new Error("Invalid 'azp' claim");
```

Comment:
@alexisintech instead of manually verifying the `azp` since the `verifyToken` should validate it, you could pass the `permittedOrigins` as `authorizedParties` option in the `verifyToken`

Reply:
good points !!! it works [39c42f9](https://github.com/clerk/clerk-docs/pull/1242/commits/39c42f93546b85343f39c701fd9b0d4b340da2d5)